src/com/stream/FilterListElement.java

```diff
@@ -11,7 +11,7 @@ public static void main(String[] args) {
 
 		List<Integer> list = new ArrayList();// initializing list
 		// adding element into list
-		for (int i = 1; i <= 20; i++) {
+		for (int i = 1; i <= 40; i++) {
```

Comment:
this number in future can change from 40 to any number, can u declare a local variable and assign that variable in this condition.
int x = 40;
I<=x;